exam/models/exam.py

```diff
@@ -245,14 +241,6 @@ def set_cancel(self):
         '''Method to set state to cancel'''
         self.write({'state': 'cancelled'})
 
-    @api.multi
-    def _validate_date(self):
-        '''Method to check start date should be less than end date'''
-        for exm in self:
-            if exm.start_date > exm.end_date:
-                return False
-        return True
-
```

Comment:
any reason for removing? Everytime you make PR, better you mention what activities are done